packages/client/src/router/index.js

```diff
@@ -110,9 +119,18 @@ const routes = [
       {
         path: '/agencies',
         name: 'agencies',
-        component: () => import('../views/Agencies.vue'),
```

Comment:
Similar to the above comment – rather than a duplicate route, where `/agencies` and `/teams` are hooked up to the same `Teams.vue` component, we could add a [redirect](https://router.vuejs.org/guide/essentials/redirect-and-alias#Redirect) so that any links pointing to the old `/agencies` URLs are directed to `/teams`.

I'm not sure offhand if Vue redirects inform the browser of anything beyond the actual redirection behavior – e.g. like how a 301 HTTP status code tells user-agents/browsers to update cached links and future requests – but at the very least, a redirect would help prevent new links/references to the `/agencies` route (which is effectively deprecated by these changes) from being created.

Reply:
refactored to dynamically apply a redirect for agencies and tenants.

Reply:
Hopefully I didn't miss the point of your comment. I implemented the redirect. Regarding the duplicate route, I assume the path being dynamic would also mean you wouldn't need a redirect.

Reply:
The updates look good to me. 

For posterity, by "duplicate", I was referring to the presence of code for both the `/agencies` and `/teams` routes existing in this PR, with `/agencies` (as the legacy route) being updated to use `Teams.vue`, resulting in both routes pointing to the same underlying view. Adding the redirect is effectively the same behavior, but with the enhancement of ensuring the old route won't continue to be referenced in new links.

Reply:
Sounds good.